app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java

```diff
@@ -46,6 +48,8 @@
 public class HabitListFragment extends ListFragment<Habit> {
 
     private static final String TAG = "HabitListFragment";
+    private static final String CANCELLED_MESSAGE = "Cancelled swap";
+    private static final String SWAP_MESSAGE = "Please select another Habit to swap with.";
```

Comment:
Another thing, we don't have punctuation **anywhere else** in our toast messages. It would look better if the period wasn't here.